dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -93,7 +93,8 @@ def exit_survey(workshop, teacher, enrollment)
       CDO.warn "Skipping attempt to send a duplicate workshop survey email. Enrollment: #{enrollment.id}"
       return
     end
-    enrollment.update!(survey_sent_at: Time.zone.now)
+    # Skip school validation to allow legacy enrollments before those fields were required to update.
```

Comment:
suggest: "Skip school validation to allow legacy enrollments (from before those fields were required) to update."